src/Compilers/CSharp/Test/Emit/Attributes/EmitTestStrongNameProvider.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+// The use of SigningTestHelpers makes this necessary for now. Fix is tracked by https://github.com/dotnet/roslyn/issues/25228
+#if NET461
```

Comment:
FYI @333fred. I had to add the condition back for this particular type. The `SigningTestHeplers` usage is too difficult to unwind in this PR. Filed a follow up issue to make this type portable friendly and remove this `#if`. I've already made a bit of progress here but given the size of this PR and the one to fix this type decided it was best to split them up.